vaadin-cdi/src/test/java/com/vaadin/cdi/CdiVaadinServletTest.java

```diff
@@ -64,6 +67,8 @@ public void setUp() throws ServletException {
                 .thenReturn(provider);
         Mockito.when(servletContext.getAttribute(Lookup.class.getName()))
                 .thenReturn(lookup);
+        Mockito.when(servletContext.getAttribute(Lookup.class.getName()))
```

Comment:
This is a duplicate of the previous line. Needs a second Supplier argument?

Reply:
No, should be removed. Good catch!